cmd/common/image/helpers/Makefile.build

```diff
@@ -41,7 +41,7 @@ wasm: $(WASM)
 else ifeq (rs, $(patsubst %.rs,rs,$(WASM)))
 wasm: $(WASM)
 	cd $(dir $(WASM)) && \
-	CARGO_TARGET_DIR=$(OUTPUTDIR) cargo build --target wasm32-wasip1 --release && \
+	CARGO_TARGET_DIR=$(OUTPUTDIR) cargo --color always build --target wasm32-wasip1 --release && \
```

Comment:
Cargo does not detect whether the terminal supports colours because the container runs without tty.

What about detecting if the terminal support colours with this?
https://github.com/jwalton/go-supportscolor/blob/master/example_test.go

In this way, colours would remain disabled when using `ig image build ... > build.log` but would be enabled when writing the output on the terminal.